highfidelity/hifi
# Fix dde visibility

https://highfidelity.fogbugz.com/f/cases/7789/Settings-Menus-Do-not-Update

Test plan:
- Enable Developer menus
- On tablet/toolbar, go to Menu > Developer > Avatar > Face Tracking. Notice that "Use Camera" is selected and has the options {"Binary Eyelid Control", "Couple Eyelids", "Use Audio for Mouth"... etc.}
- Select "None"
- Close the dialog or go home on tablet and close the tablet.
- Go to Menu > Developer > Avatar > Face Tracking 
- Select "Use Camera". 
- Make sure that the settings for "Use Camera" appears
